Content.Server/Kudzu/SpreaderSystem.cs

```diff
@@ -0,0 +1,131 @@
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.Atmos.Components;
+using Content.Shared.Atmos;
+using Content.Shared.Movement.EntitySystems;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.Random;
+
+namespace Content.Server.Kudzu;
+
+// Future work includes making the growths per interval thing not global, but instead per "group"
+public class SpreaderSystem : EntitySystem
```

Comment:
yeah this whole class seems very kudzu-specific still since it can't handle the case of having multiple different spreader groups that are independent of eachother, but that is fine for now. foam/smoke handles this by having an 'area effect inception' that handles the group behavior

Reply:
Noted.